### ISSUE TITLE
Server broken after workshop addon update yesterday

### ISSUE THREAD

**Issue by MisterRando** [FPtje/DarkRP#2909]
[//]: # (DO NOT REMOVE THIS TEMPLATE.)
[//]: # (GITHUB ISSUES ARE **NOT** FOR HELP)
[//]: # (GO TO THE FORUMS INSTEAD: https://forum.darkrp.com/ )

### Description of the bug
[//]: # (Describe the issue as accurately as possible)

When the server starts I now get an error and DarkRP does not load since the workshop addon was updated.

### How to make the bug happen
[//]: # (Knowing how to make a bug happen allows the developers to figure out)
[//]: # (what the cause of the problem is and whether a certain fix solves it)

It happens when the server starts and tries to load DarkRP.

### Lua errors
[//]: # (Note that errors on server startup are more important than other ones)
[//]: # (because they can cause other errors. Always look in the startup log of the server!)

New error I have never seen before this last workshop addon update. This is the first error in the server console as well as the game console.

[DarkRP] gamemodes/darkrp/gamemode/libraries/simplerr.lua:530: [ERROR] A runtime error has occurred in "gamemodes/darkrp/gamemode/libraries/tablecheck.lua" on line 157.
The best help I can give you is this:
'IsEmpty' is being called like it is a function, but in reality it does not exist (is nil).
Hints:
- You are doing either stuff.something(<otherstuff>) or stuff:something(<otherstuff>). The 'something' here does not exist.
The responsibility for the error above lies with (the authors of) one (or more) of these files:
1. gamemodes/darkrp/gamemode/libraries/tablecheck.lua on line 157
2. gamemodes/darkrp/gamemode/libraries/fn.lua on line 185
3. gamemodes/darkrp/gamemode/libraries/tablecheck.lua on line 67
4. gamemodes/darkrp/gamemode/libraries/tablecheck.lua on line 85
5. gamemodes/darkrp/gamemode/libraries/fn.lua on line 174
6. gamemodes/darkrp/gamemode/modules/base/sh_createitems.lua on line 416
7. addons/darkrpmodification-master/lua/darkrp_customthings/jobs.lua on line 50
8. gamemodes/darkrpCouldn't Load Init Script: 'darkrp/gamemode/init.lua'

### Why the developer of DarkRP is responsible for this issue
[//]: # (By posting on Github, you ask the developers of DarkRP to solve the problem)
[//]: # (It may seem obvious to you, but you have to make clear why they are the right people to look at it)

I have not changed or modified anything on my server in the past few days. I have not added or removed any addons. The DarkRP addon on the workshop updated yesterday and after my server restarted since that update it has been broken and unplayable due to DarkRP not being able to load becuase of the errors that started after the addon was updated and after my server performed its daily restart.

ERROR LINKS
Server - https://pastebin.com/w76Vz9gd
Client - https://pastebin.com/mBCkdmjv

[clientside_errors.txt](https://github.com/FPtje/DarkRP/files/3224903/clientside_errors.txt)
[lua_errors_server.txt](https://github.com/FPtje/DarkRP/files/3224904/lua_errors_server.txt)
[first main error.txt](https://github.com/FPtje/DarkRP/files/3224905/first.main.error.txt)

Long story short: The DarkRP workshop addon was updated. That is the only thing that has changed since this problem started occurring. I have not touched a thing in days so do not believe I caused this.

**Comment by FPtje** [comment]
`IsEmpty` error means your server is out of date. Please update your server.

**Comment by Bo98** [comment]
Maybe it's worth adding that GMod version check on startup, to at least give an informative error.
